charts/hawkbit-update-server/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-version: 0.6.1
+version: 0.7.0
```

Comment:
removing the user / passwords vars is a breaking change so we should go for version 1.0.0